osx/build-ruby

```diff
@@ -325,12 +331,15 @@ if [[ "$GEMFILE" != "" ]]; then
 
 	# Install Bundler, either from cache or directly.
 	if [[ -e "$RUNTIME_DIR/vendor/cache/bundler-$BUNDLER_VERSION.gem" ]]; then
-		run "$TMPBUILDROOT/bin/gem" install "$RUNTIME_DIR/vendor/cache/bundler-$BUNDLER_VERSION.gem" --no-rdoc --no-ri
+		echo "install Bundler from cache"
+		run "$TMPBUILDROOT/bin/gem" install "$RUNTIME_DIR/vendor/cache/bundler-$BUNDLER_VERSION.gem" --no-document
 	else
-		run "$TMPBUILDROOT/bin/gem" install bundler -v $BUNDLER_VERSION --no-rdoc --no-ri
+		echo "install Bundler directly"
+		run "$TMPBUILDROOT/bin/gem" install bundler -v $BUNDLER_VERSION --no-document
 		run mkdir -p "$RUNTIME_DIR/vendor/cache"
 		run cp "$TMPBUILDROOT"/lib/ruby/gems/$RUBY_COMPAT_VERSION/cache/bundler-$BUNDLER_VERSION.gem \
 			"$RUNTIME_DIR/vendor/cache/"
+		run "$TMPBUILDROOT/bin/gem" install "$RUNTIME_DIR/vendor/cache/bundler-$BUNDLER_VERSION.gem" --no-document
```

Comment:
I had issues with bundler not being installed, because this was defaulting to the system ruby ( I use rbenv ) so I had to install the correct version of bundler, on my 2.6.8 local version of ruby

some of my attempts to install the right version of bundler against the system version of ruby are evident in the CI runs https://github.com/phusion/traveling-ruby/pull/133/files#diff-62587956f943bb2503db7bc6dd27d0d888074a1c0ecaab3f570ad611aff0f7bbR33